pytest-selenium/pages/content.py

```diff
@@ -1,20 +1,36 @@
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support import expected_conditions as expected
 
+
 from pages.base import Page
 from regions.base import Region
+from regions.content_item import ContentItem
 
 
 class Content(Page):
     URL_TEMPLATE = "/books/{book_slug}/pages/{page_slug}"
-
+    _title_locator = (By.TAG_NAME, "title")
     _body_locator = (By.TAG_NAME, "body")
     _main_content_locator = (By.CSS_SELECTOR, "h1")
+    _next_locator = (By.CSS_SELECTOR, "[aria-label='Next Page']")
+    _previous_locator = (By.CSS_SELECTOR, "[aria-label='Previous Page']")
 
     @property
     def loaded(self):
         return self.find_element(*self._body_locator).get_attribute("data-rex-loaded")
 
+    @property
+    def title(self):
+        return self.find_element(*self._title_locator)
```

Comment:
titiles apply to all pages, not just content pages, this could go in the super class

Reply:
created a helper function for title. I shall add this as super class when we have more pages coming into Rex